prime-router/src/main/kotlin/azure/batch/BatchDeciderFunction.kt

```diff
@@ -56,7 +56,12 @@ class BatchDeciderFunction(private val workflowEngine: WorkflowEngine = Workflow
                                 BatchConstants.Queue.COVID_BATCH_QUEUE
                             }
 
-                            workflowEngine.queue.sendMessageToQueue(event, queueName)
+                            val delay = rec.timing!!.timeBetweenBatches
+                            if (delay.isZero) {
+                                workflowEngine.queue.sendMessageToQueue(event, queueName)
```

Comment:
We can just pass the absolute value (including if 0) and that will reduce a lot of the code?